### ISSUE TITLE
dmem shrink

### ISSUE THREAD

**Issue by ppshirvani** [makaimann/ride-core-demo#1]
Are you sure this change "In dmem.v: Shrink the data memory" has been done in this demo?  Looks like the original (start) file is the same as the later one.

**Comment by makaimann** [comment]
Hello!

Sorry for the slow response, I didn't get a notification for some reason. I actually meant to email you about this exact change, but I dropped the ball. 

I've been making some improvements to the demo based on feedback, including this "Shrink the data memory" change. However, I've been doing terrible git rebases and force pushes to rewrite the git history. My reason for this is that I want very few commits to keep the demo clean.

Basically, I think this change has been done in the current master branch, but the version of the demo you have probably does not have it, and it's difficult to tell because of the history rewrites.

Note: the demo should run fine with the version you have, it will just be faster with the latest.

To get the latest master, run:
```
git fetch --all
git reset --hard origin/master
```

However, you will lose whatever version you had before entirely. I've checked that the verification results are as expected with this latest version though.

Hopefully that answers your question!
